src/widgets/speakers/ui/speakers.module.scss

```diff
@@ -0,0 +1,56 @@
+.speakers {
+  display: flex;
+  gap: 50px;
+  background-color: $color-gray-100;
+
+  .info {
+    width: 640px;
+    font-weight: $font-weight-medium;
+    color: $color-black;
```

Comment:
I could be wrong, but it looks like they don't affect anything.

Reply:
nice catch, removed